script/src/validate-monorepo/validation/index.ts

```diff
@@ -12,10 +12,12 @@ export type ValidationIssue =
 
 export async function* validateMonorepo(): AsyncGenerator<ValidationIssue> {
   const root = join(__dirname, '../../../..');
+  const apps = await readdir(join(root, 'apps'));
+  const appPackages = flatten(await Promise.all(apps.map(readdir)));
```

Comment:
Is this different from `Array#flat`?
```js
> [[1,2],[3,4]].flat()
[ 1, 2, 3, 4 ]
```

Reply:
Oh, for some reason I couldn't find `Array#flat`. Thanks!